src/components/modals/SupportRequestModal.js

```diff
@@ -0,0 +1,194 @@
+import { Component } from 'react';
+import { div, form, input, label, textarea, hh, select, option } from 'react-hyperscript-helpers';
+import { SupportRequestBaseModal } from '../SupportRequestBaseModal';
+import { Alert } from '../Alert';
+import { Support, User } from '../../libs/ajax';
+import { Storage } from "../../libs/storage";
+import { convertCompilerOptionsFromJson } from 'typescript';
+
+export const SupportRequestModal = hh(class SupportRequestModal extends Component {
+  constructor(props) {
+    super(props);
+    const isLogged = Storage.userIsLogged();
+    let name = isLogged ? Storage.getCurrentUser().displayName : '';
+    let first_name = isLogged ? ", " + name.split(" ")[0] : '';
+    let email = isLogged ? Storage.getCurrentUser().email : '';
+    let height = isLogged ? "550px": '700px';
+    let top = isLogged ? '400px': '1';
```

Comment:
All of these `let`s can be `const`s. Also, you could avoid this altogether and just put the ternary expressions into the state object directly.

Reply:
When I put them in the state objects I get this error "Unexpected use of 'name'  no-restricted-globals"

Reply:
I think that's because you're using `name` to calculate `first_name`. I think this should be easy to resolve.

Reply:
Try this:
```
    this.state = {
      name: Storage.userIsLogged() ? Storage.getCurrentUser().displayName : '',
      isLogged: Storage.userIsLogged(),
      type: 'question',
      subject: '',
      description: '',
      attachment: '',
      email: Storage.userIsLogged() ? Storage.getCurrentUser().email : '',
      first_name: Storage.userIsLogged() ? Storage.getCurrentUser().displayName.split(' ')[0] : '',
      height: Storage.userIsLogged() ? '550px': '700px',
      top: Storage.userIsLogged() ? '400px': '1',
      valid: Storage.userIsLogged()
    };
```